code/modules/food_and_drink/snacks.dm

```diff
@@ -2189,6 +2205,29 @@ ABSTRACT_TYPE(/obj/item/reagent_containers/food/snacks/soup)
 		else
 			src.UpdateOverlays(null, "dip")
 
+/obj/item/reagent_containers/food/snacks/churro
+	name = "churro"
+	desc = "It's like a donut, but long."
+	icon = 'icons/obj/foodNdrink/food_snacks.dmi'
+	icon_state = "churro"
+	bites_left = 3
+	heal_amt = 1
+	food_effects = list("food_energized")
+
+	New()
+		..()
+		src.pixel_x = rand(-6, 6)
+		src.pixel_y = rand(-6, 6)
```

Comment:
isn't there an obj var for this

Reply:
for whatever's going on in `New()`? i've got no idea so i'm preserving that for the refactor